src/Rakaposhi.Core/Rakaposhi.API.Core/Controllers/TransTypeController.cs

```diff
@@ -0,0 +1,98 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Rakaposhi.Business.Core.DataObjects;
+using Rakaposhi.Business.Core.Services;
+
+namespace Rakaposhi.API.Core.Controllers
+{
+    [Route(Global.APICONTROLLER)]
+    [ApiController]
+    public class TransTypeController : Controller
+    {
+        private TransTypeService _transTypeService;
+
+        public TransTypeController(TransTypeService transTypeService)
+        {
+            _transTypeService = transTypeService;
+        }
+
+        [HttpPost]
+        public ActionResult Create([FromBody] TransType transactionTypes)
```

Comment:
What if we set augment name transactionType rather than transactionTypes ? because it create single record.